lombok.config

```diff
@@ -0,0 +1,3 @@
+# Copy the Qualifier annotation from the instance variables to the constructor
+# see https://github.com/rzwitserloot/lombok/issues/745
+lombok.copyableAnnotations += org.springframework.beans.factory.annotation.Qualifier
```

Comment:
Are you sure it won't break anything in the existing code at all?

Reply:
yes.  it is not the first xm ms (and ms in a company where i work) where i added this feature.

Reply:
@ibilous  i mean it's change exists behavior and can add annotation in existing code when it's annotation not expecting

Reply:
no it's not.  What does it do: for example u  want to add this annotation for the field and u  use @RequiredArgsConstructor for convenience. In this case the annotation @Qualifier won't be added to the constructor. In this case u cannot use RequiredArgsConstructor and must write constructor sometimes for ~10 parameters. But if u use this lombok  config it do it for u. It adds annotation Qualifier to the constructor that is built by RequiredArgsConstructor, It won't add the annotation where it is not expected.